service/service.go

```diff
@@ -1095,13 +1095,7 @@ func (s *service) GetNodeLabelsWithName(nodeName string) (map[string]string, err
 func (s *service) PatchNodeLabels(add map[string]string, remove []string) error {
 	log := logging.GetLogger()
 
-	k8sclientset, err := k8sutils.CreateKubeClientSet(s.opts.KubeConfigPath)
-	if err != nil {
-		log.Errorf("init client failed: '%s'", err.Error())
-		return err
-	}
-
-	node, err := k8sclientset.CoreV1().Nodes().Get(context.TODO(), s.nodeID, v1.GetOptions{})
+	node, err := s.k8sclient.CoreV1().Nodes().Get(context.TODO(), s.nodeID, v1.GetOptions{})
```

Comment:
GetNodeLabels also has this issue. I copied from that function so it may also be wrong.